tests/phpunit/integration/Core/Util/Auto_UpdatesTest.php

```diff
@@ -44,35 +45,30 @@ public function test_sitekit_autoupdate_forced() {
 		remove_filter( 'auto_update_plugin', '__return_false' );
 	}
 
+	/**
+	 * Forced auto updates are available for WP >= 5.6 only, so we test
+	 * for a function only available in WordPress 5.6 or greater.
+	 *
+	 * @requires function wp_is_auto_update_forced_for_item
+	 */
 	public function test_sitekit_autoupdates_disabled() {
-		// Forced auto updates are available for WP >= 5.6 only.
-		if ( ! self::auto_updates_available() ) {
-			return;
-		}
-
 		$this->assertFalse( Auto_Updates::is_sitekit_autoupdates_enabled() );
 
 		update_site_option( 'auto_update_plugins', array( 'other-plugin.php' ) );
 
 		$this->assertFalse( Auto_Updates::is_sitekit_autoupdates_enabled() );
 	}
 
+	/**
+	 * Forced auto updates are available for WP >= 5.6 only, so we test
+	 * for a function only available in WordPress 5.6 or greater.
+	 *
+	 * @requires function wp_is_auto_update_forced_for_item
+	 */
 	public function test_sitekit_autoupdates_enabled() {
-		// Forced auto updates are available for WP >= 5.6 only.
-		if ( ! self::auto_updates_available() ) {
-			return;
-		}
-
 		update_site_option( 'auto_update_plugins', array( 'other-plugin.php', GOOGLESITEKIT_PLUGIN_BASENAME ) );
 
 		$this->assertTrue( Auto_Updates::is_sitekit_autoupdates_enabled() );
 	}
-
-	/**
-	 * Verify if auto-updates are available for the current WordPress version.
-	 */
-	private function auto_updates_available() {
-		return version_compare( get_bloginfo( 'version' ), '5.5', '>=' );
```

Comment:
According to this comparison, the tests that were using this helper should also run for WP 5.5, which is where it would be useful to require `wp_is_auto_update_enabled_for_type`. I think some of the comments throughout made that a bit confusing.

Reply:
Argh, yes, I didn't realise. Will update 👍🏻